### DISCUSSION TITLE
What is the function of  parameter Preview Settings In Perlin Noise Gradient component

### DISCUSSION THREAD

**Discussion by qq260105064** [o3de/o3de#7881] [Q&A]
**Describe the bug**
There is no change in vegetation after modifying this parameter.

**Steps to reproduce**
1. Modify parameter Preview Settings in component Perlin Noise Gradient

**Expected behavior**
Want to know the function of parameter Preview Settings.

**Desktop/Device (please complete the following information):**
 - Device: [e.g. PC, Mac, iPhone, Samsung] 
 - OS: [e.g. Windows, macOS, iOS, Android]
 - Version [e.g. 10, Bug Sur, Oreo]
 - CPU [e.g. Intel I9-9900k , Ryzen 5900x, ]
 - GPU [AMD 6800 XT, NVidia RTX 3090]
 - Memory [e.g. 16GB]

**Additional context**

**Comment by monroegm** [comment]
@qq260105064 Can you ask this question in the #sig-content discord channel?
